assets/properties-panel.css

```diff
@@ -98,6 +99,10 @@
   --dropdown-item-hover-background-color: var(--color-grey-225-10-95);
   --dropdown-separator-background-color: var(--color-grey-225-10-75);
 
+  --svg-stroke-color: var(--color-grey-225-10-35);
```

Comment:
Can we come up with a semantic name? Or is this the color of all SVG strokes?

Reply:
changed to `--feel-active/inactive-color`